routes/usersRouter.js

```diff
@@ -0,0 +1,87 @@
+require('dotenv').config()
+const jwt = require('jsonwebtoken')
+const {JWT_SECRET = "neverTell"} = process.env
+const express = require('express');
+const usersRouter = express.Router();
+const { requireUser } = require('./utils');
+
+const {
+  getUserByUsername,
+  getUser,
+  getAllUsers,
+} = require('../db/users');
+
+const {
+	createUser,
+} = require('../db/index');
+
+usersRouter.get('/', async(req, res, next) => {
```

Comment:
REQUIRED - Should be admin-only access. Either secure this route or "stash" the code in a new branch for now.